cmd/meroxa/root/resources/create.go

```diff
@@ -130,14 +131,28 @@ func (c *Create) Execute(ctx context.Context) error {
 		}
 	}
 
+	if sshURL := c.flags.SSHURL; sshURL != "" {
+		input.SSHTunnel = &meroxa.ResourceSSHTunnelInput{
+			Address: sshURL,
+		}
+	}
+
 	c.logger.Infof(ctx, "Creating %q resource...", input.Type)
 
 	res, err := c.client.CreateResource(ctx, &input)
 	if err != nil {
 		return err
 	}
 
-	c.logger.Infof(ctx, "%q resource successfully created!", res.Name)
+	if tun := res.SSHTunnel; tun == nil {
+		c.logger.Infof(ctx, "%q resource successfully created!", res.Name)
+	} else {
+		c.logger.Infof(ctx, "%q resource created!", res.Name)
```

Comment:
This output really should be the same as above (`c.logger.Infof(ctx, "%q resource successfully created!", res.Name)`) and then outside the conditional in consequence.

In other words, remove the `if`/`else` clause, always show that confirmation message once resource is created and then only add the code for the `SSHTunnel`.

Reply:
The message is intentionally omitting "successful". Perhaps there is some misunderstanding of how ssh tunnel works. If a resource has an ssh tunnel, the validation of the resource will be async. So the resource is created but it's not successful until validation goes through until later (customers will need to follow the instruction to put the public key in the bastion etc.). Does it make sense?

Reply:
Perhaps there is better way to rephrase the message to avoid confusion and make the above more clear.

Reply:
I took a stab: https://github.com/meroxa/cli/pull/148/commits/7bfc76e4770fe8b1888eb144cd10e80e040b9b10. What do you think now? @raulb

Reply:
@owenthereal I think that's perfect, thank you!